src/pkg/retention/manager.go

```diff
@@ -99,3 +103,13 @@ func (d *DefaultManager) GetPolicy(ctx context.Context, id int64) (*policy.Metad
 func NewManager() Manager {
 	return &DefaultManager{}
 }
+
+func addNextScheduledTime(p *policy.Metadata) error {
+	if p.Trigger.Kind == policy.TriggerKindSchedule {
+		cron, ok := p.Trigger.Settings[policy.TriggerSettingsCron]
+		if ok && len(cron.(string)) > 0 {
+			p.Trigger.Settings[policy.TriggerSettingNextScheduledTime] = strfmt.DateTime(utils.NextSchedule(p.Trigger.Settings[policy.TriggerSettingsCron].(string), time.Now()))
```

Comment:
`utils.NextSchedule(p.Trigger.Settings[policy.TriggerSettingsCron] => cron?`

Reply:
Yes, verified on hourly, daily, weekly, and custom cron setting.